plugins/autocomplete/plugin.js

```diff
@@ -857,94 +857,165 @@
 		 * For example, the position of the bottom end of the caret.
 		 */
 		setPosition: function( rect ) {
-			var editor = this.editor,
-				viewHeight = this.element.getSize( 'height' ),
-				editable = editor.editable(),
-				// Bounding rect where the view should fit (visible editor viewport).
-				editorViewportRect;
+			var documentWindow = this.element.getWindow(),
+				windowRect = documentWindow.getViewPaneSize(),
+				top = setVerticalPosition( {
```

Comment:
Well, it doesn't really set the position it just calculates it returning `top` placement. The position is set in [line 875 with `setStyle`](https://github.com/ckeditor/ckeditor4/pull/4266/files#diff-679aa38255fe9479e45442e51e5b5daca8bc8df2e3b06102929c33fd2e7b1853R875).